101-synapse-poc/azuredeploy.json

```diff
@@ -0,0 +1,621 @@
+                    "companyTla": { "value": "[parameters('companyTla')]" },
+                    "deploymentType": { "value": "[parameters('deploymentType')]" },
+                    "TIME_ZONE": { "value": "[parameters('TIME_ZONE')]" },
+                    "PauseTime": { "value": "[parameters('PauseTime')]" },
+                    "location": { "value": "[parameters('location')]" }
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "logicAppResumeDeployment",
+            "properties": {
+                "mode": "Incremental",
+                "templatelink": {
+                    "uri": "https://raw.githubusercontent.com/JamJarchitect/azure-quickstart-templates/master/101-synapse-poc/nestedtemplates/resumetemplate.json"
+                },
+                "parameters": {
+                    "logicAppName": { "value": "[parameters('logicAppResumeName')]" },
+                    "Frequency": { "value": "[parameters('Frequency')]" },
+                    "companyTla": { "value": "[parameters('companyTla')]" },
+                    "deploymentType": { "value": "[parameters('deploymentType')]" },
+                    "TIME_ZONE": { "value": "[parameters('TIME_ZONE')]" },
+                    "ResumeTime": { "value": "[parameters('ResumeTime')]" },
+                    "location": { "value": "[parameters('location')]" }
+                }
+            }
+        },
+        {
+            "apiVersion": "2019-06-01-preview",
+            "name": "[variables('workspaceName')]",
+            "location": "[parameters('location')]",
+            "type": "Microsoft.Synapse/workspaces",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "defaultDataLakeStorage": {
+                    "accountUrl": "[variables('defaultDataLakeStorageAccountUrl')]",
+                    "filesystem": "[variables('dlsFsName')]"
+                },
+                "sqlAdministratorLogin": "[parameters('sqlAdministratorLogin')]",
+                "sqlAdministratorLoginPassword": "[parameters('sqlAdministratorLoginPassword')]",
+                "managedVirtualNetwork": "default"
+            },
+            "resources": [
+                {
+                    "condition": "[equals(parameters('allowAllConnections'),'true')]",
+                    "apiVersion": "2019-06-01-preview",
+                    "dependsOn": [
+                        "[variables('workspaceName')]"
+                    ],
+                    "location": "[parameters('location')]",
+                    "name": "allowAll",
+                    "properties": {
+                        "startIpAddress": "0.0.0.0",
+                        "endIpAddress": "255.255.255.255"
+                    },
+                    "type": "firewallrules"
```

Comment:
check the bp guide for sort order of the properties... https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties